snippets/base/forms.py

```diff
@@ -88,7 +88,6 @@ class Media:
         js = filter(None, [
             'js/lib/jquery-2.2.1.min.js',
             'js/lib/nunjucks{0}.js'.format('-dev' if settings.DEBUG else ''),
-            'templates/compiled.js' if not settings.DEBUG else None,
```

Comment:
Since this was the only item in the list that might be None, I think you should be able to remove the filter on line 88 as well.